core/keysign/preSignedInputData/ripple.ts

```
@@ -1,20 +1,19 @@
-import Long from "long";
+import { assertField } from '@lib/utils/record/assertField'
+import { TW } from '@trustwallet/wallet-core'
+import Long from 'long'
 
-import {
-  GetPreSignedInputDataInput,
-  PreSignedInputDataResolver,
-} from "./PreSignedInputDataResolver";
+import { 
+  PreSignedInputDataResolver, 
+} from './PreSignedInputDataResolver'
```

ESLint

[error] Replace `⏎··PreSignedInputDataResolver,·⏎` with `PreSignedInputDataResolver·`

(prettier/prettier)